services/worker/src/worker/config.py

```diff
@@ -19,9 +19,9 @@
 WORKER_HEARTBEAT_INTERVAL_SECONDS = 60
 WORKER_KILL_LONG_JOB_INTERVAL_SECONDS = 60
 WORKER_KILL_ZOMBIES_INTERVAL_SECONDS = 10 * 60
-WORKER_MAX_DISK_USAGE_PCT = 90
+WORKER_MAX_DISK_USAGE_PCT = 70
```

Comment:
I think we should change this in another PR

Reply:
Sorry, it was a mistake, I removed it from the PR